force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -162,17 +201,18 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static String repeatString(String inputString, Integer repeatCount) {
         // Initialize the result String
+        String resultString = '';
 
         // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
-
-            // If it is not the last iteration, add a semicolon to separate the strings
+        for (Integer i = 0; i < repeatCount; i++) {
+            resultString += inputString;
+            if (i != repeatCount - 1) {
+                resultString += '; ';
+            }
```

Comment:
Is there a method in the `String` class that performs this function? 

Otherwise, this is just fine.